lib/foods.js

```diff
@@ -108,6 +111,35 @@ function handleFoodEdit () {
   replaceValue($input, id, changedAttribute)
 };
 
+function enterEditPress (){
+  if (event.which == 13) {
```

Comment:
I'm convinced that there's a simpler way to do this, but I have no idea what it is. Thanks for being so perseverant on this.